xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -328,6 +328,11 @@ const std::string& CMusicInfoTag::GetSongVideoURL() const
   return m_songVideoURL;
 }
 
+const std::map<int, chapterDetails>& CMusicInfoTag::GetChapterMarks() const
+{
+  return m_chapters;
+}
+
```

Comment:
Because impl. is trivial, I would make this method definition inline in the header.

Reply:
Yeah, 95% of that file is trivial stuff like that.  However, I'd like to keep all the implementations together rather than defining new ones in the header.

Reply:
Up to you of course.